lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -11,60 +9,8 @@ import ErrorDisplay from './ErrorDisplay';
  */
 
 /**
- * Common structure for LTI launch error dialogs.
- *
- * This is a "full-screen", non-cancelable dialog consisting of:
- *
- *  - An explanation of what went wrong and how to resolve it
- *  - An optional "Try again" button
- *  - An optional detailed error message for use when contacting support
- *
- * @param {object} props
- * @param {boolean} props.busy
- * @param {Children} props.children
- * @param {ErrorLike|null} [props.error]
- * @param {() => void} [props.onRetry]
- * @param {string} [props.retryLabel]
- * @param {string} [props.title]
+ * @typedef {import('./ErrorModal').ErrorModalProps} BaseDialogProps
  */
-function BaseDialog({
```

Comment:
This is no longer needed; some of the needs that this served — handling `busy`, `retryLabel`, `onRetry`, etc. — have been integrated into the new `ErrorModal` so this component can use it directly without an intermediary.